components/raftstore/src/store/fsm/apply.rs

```diff
@@ -1547,6 +1553,10 @@ where
             AdminCmdType::PrepareMerge => self.exec_prepare_merge(ctx, request),
             AdminCmdType::CommitMerge => self.exec_commit_merge(ctx, request),
             AdminCmdType::RollbackMerge => self.exec_rollback_merge(ctx, request),
+            AdminCmdType::PrepareFlashback => {
+                self.set_flashback_state(ctx, FlashbackState::Processing)
+            }
+            AdminCmdType::FinishFlashback => self.set_flashback_state(ctx, FlashbackState::Finish),
```

Comment:
Why not merge these two into one `AdminCmdType::SetFlashbackState`?

Reply:
Do u mean that we can set the FlashbackState by setting a field in the admin request that serves to synchronize the current Flashback state?

Reply:
Oh, I get it we use different admin cmd types to write different states.